clippy_lints/src/casts/mod.rs

```diff
@@ -277,23 +283,31 @@ declare_clippy_lint! {
     ///
     /// ### Example
     /// ```rust
-    /// // Bad: fn1 is cast as `usize`
+    /// // fn1 is cast as `usize`
     /// fn fn1() -> u16 {
     ///     1
     /// };
-    /// let _ = fn1 as usize;
+    /// # let _ =
+    /// fn1 as usize;
+    /// ```
```

Comment:
While this works, I believe this will look weird, when the lint output is added to the lint list. Then the output would contain a `let _ =` which is not present in the example.

 ```suggestion
    /// # let _ = fn1 as usize;
    /// ```
```

Reply:
Good call, I probably did that in other places. I'll get that fixed.